features/step_definitions/fixtures_steps.rb

```diff
@@ -44,9 +44,8 @@
 end
 
 Given(/^"([^"]*)" has been added as an MenuItem to "([^"]*)"$/) do |child, parent|
-  #binding.pry
   menu = Menu.find_by(title: parent)
   menu_item = MenuItem.find_by(name: child)
-  menu.menu_items << menu_item
+  MenuItemsMenu.create(daily_stock: 10, menu_item: menu_item, menu: menu)
```

Comment:
We were failing with 'menu.menu_items << menu_item', since that statement was resulting in the creation of the join record in menu_items_menus - which was failing because the menu_items_menu model validates that daily_stock is a number and non-negative.  This change worked around that problem.  Your suggestion may work ..... will need to test that.